guide/_sidebar.md

```diff
@@ -13,6 +13,7 @@
 	* [Configuration files](/creating-your-bot/configuration-files)
 	* [Adding more commands](/creating-your-bot/adding-more-commands)
 	* [Commands with user input](/creating-your-bot/commands-with-user-input)
+	* [Parsing mention arguments](/creating-your-bot/parsing-mention-arguments)
```

Comment:
I don't think this should be put under this section of the sidebar, because when people read the guide in order, they'll come to this page and think it's necessary for them to read when it's a bit more advanced than the beginner level feel of the rest of the pages before it.

It doesn't exactly fit under "Popular topics", but maybe we can take the "Common questions" and "Miscellaneous examples" sections, put them under a section called "Miscellaneous", and stick this under it?